modules/tinymce/src/plugins/table/main/ts/api/Events.ts

```diff
@@ -11,12 +11,12 @@ import Editor from 'tinymce/core/api/Editor';
 const fireNewRow = (editor: Editor, row: HTMLElement) => editor.fire('newrow', { node: row });
 const fireNewCell = (editor: Editor, cell: HTMLElement) => editor.fire('newcell', { node: cell });
 
-const fireObjectResizeStart = (editor: Editor, target: HTMLElement, width: number, height: number) => {
-  editor.fire('ObjectResizeStart', { target, width, height });
+const fireObjectResizeStart = (editor: Editor, target: HTMLElement, width: number, height: number, origin: string) => {
+  editor.fire('ObjectResizeStart', { target, width, height, origin });
```

Comment:
Is it okay that we're putting all of the extra info into a single string parameter? It looked, above, as though there were two different parameters being condensed into this string. Should we try to hold onto the structure?

Reply:
As discussed during standup, this isn't actually 2 entirely separate things as we can't have for example `bar-se` so it needs to be combined to be meaningful. As such we decided it should remain as it is now.